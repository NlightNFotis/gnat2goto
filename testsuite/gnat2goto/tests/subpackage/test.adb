with Other; use Other;
use Other.SubOther;

procedure Test is
   Length : Other.Unsigned_8 := 4;
   Width_1 : Other.SubOther.Unsigned_16 := 10;
   Width_2 : Other.SubOther.Unsigned_16 := 11;
begin

   pragma Assert (Width_1 > Width_2);

end Test;
