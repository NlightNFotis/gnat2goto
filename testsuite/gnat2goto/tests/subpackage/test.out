[1] file test.adb line 10 assertion: FAILURE
VERIFICATION FAILED
